modules/features/manatime_feature_clients/manatime_feature_clients.info

```diff
@@ -0,0 +1,23 @@
+name = manatime_feature_clients
```

Comment:
The _human_ name should be something more easy to understand, like "Manatime clients feature".